python/ray/tests/test_client.py

```diff
@@ -6,9 +6,12 @@
 import threading
 import time
 from unittest.mock import Mock, patch
+from typing import Type
 
 import numpy as np
 import pytest
+from pydantic import BaseModel as BaseModelV2
+from pydantic.v1 import BaseModel as BaseModelV1
```

Comment:
does this imply that the test suite will only run on pydantic v2?

if yes, does it make sense to drop pydantic v1?

Reply:
ah nvm, just spotted `EXPECTED_PYDANTIC_VERSION=1.10.12` in the test suite

Reply:
We'll run all the Ray tests on Pydantic 2, and we'll also run the Serve tests with Pydantic 1.